### ISSUE TITLE
[Windows Arm64] Add build-only Arm64 CI to flutter/packages

### ISSUE THREAD

**Issue by loic-sharma** [flutter/flutter#129813]
## Background

Flutter is being updated to support Windows Arm64: https://flutter.dev/go/windows-arm64

## Work

* [ ] Add build-only Arm64 CI to flutter/packages, mirroring the way we handle multi-arch for macOS

**Comment by stuartmorgan** [comment]
Can you update when the build support actually lands in `flutter`? Once it does this should be relatively straightforward.

**Comment by stuartmorgan** [comment]
> * [ ] Promote out of `bringup`. See: [[Devicelab] Add Windows Arm64 machines to the non-staging pool #134083](https://github.com/flutter/flutter/issues/134083)

Is the current state intentional? The blocking issue was fixed but the bots are still in bringup mode.

**Comment by loic-sharma** [comment]
@stuartmorgan Yes, these tests should be able to be promoted out of `bringup` mode. There are no known blockers at this time.

Some context: infra folks expected they wouldn't add Windows Arm64 machines to the try pool due to capacity shortages. As a workaround, I switched these tests to post-submit (https://github.com/flutter/packages/pull/6166) so that they could run on the staging pool instead and could be promoted. I haven't had the chance to follow-up on that, I don't expect I'll be able to given the reprioritization.
